test/cookbooks/cerner_splunk_ingredient_test/recipes/conf_helpers.rb

```diff
@@ -0,0 +1,24 @@
+my_config = {
```

Comment:
Given that these are just ruby methods, I don't think we need to cover them with test kitchen directly. Just straight up rspec tests should suffice.